MonolithDev/gettingSongs/test_dj_queries.py

```
@@ -0,0 +1,187 @@
+        
+        for i, track in enumerate(sample_tracks, 1):
+            print(f"\n{'-'*60}")
+            print(f"TEST {i}: {track.name} by {track.artist_names}")
+            print(f"{'-'*60}")
+            
+            try:
+                result = generator.generate_queries(track)
+                
+                print(f"✅ Successfully generated {len(result.queries)} queries")
+                print(f"\nReasoning: {result.reasoning}")
+                print(f"\nSample queries:")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)